app/api/entities/entitySavingManager.ts

```diff
@@ -1,17 +1,18 @@
 import { set } from 'lodash';
-import { generateFileName } from 'api/files';
 import entities from 'api/entities/entities';
 import { EntityWithFilesSchema } from 'shared/types/entityType';
 import { UserSchema } from 'shared/types/userType';
 import { handleAttachmentInMetadataProperties, processFiles, saveFiles } from './managerFunctions';
 
-type FileAttachments = {
+export type FileAttachment = {
```

Comment:
There's a warning here about end of file export.

Reply:
Moved!